### PR TITLE
Updated mapfile reference in ppm*.properties files

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
The mapfile references in ppmBsm.properties & ppmTim.properties currently result in failure to spin up the PPM using the jpo-ode repository's docker-compose.yml file. The user can correct this easily, but preferably it should be possible to spin up the PPM without modifying these files first.

## Solution
The value of the privacy.filter.geofence.mapfile property has been changed to "/ppm_data/data/I_80.edges" in ppmBsm.properties & ppmTim.properties

## Testing
The PPM has been verified to spin up properly with these changes to the properties files.

### CODE REVIEW

**Review by drewjj — APPROVED** [review]
Looks good!

**Review by mwodahl — APPROVED** [review]
This spun up correctly for me

File: ppmTim.properties [hunk 1]
```diff
@@ -12,7 +12,7 @@ privacy.redaction.id.included=BEA10000,BEA10001
 
 # Configuration details for geofencing.
 privacy.filter.geofence=OFF
```
Comment (payneBrandon) [inline ppmTim.properties:14]: We removed the TIM message from PPM in another PR right? should we grab those changes and remove this properties file altogether?

Comment (dmccoystephenson) [reply ppmTim.properties]: That's correct, good call. I've merged dev into the feature branch and removed the `ppmTIm.properties` file.

File: .github/workflows/dockerhub.yml [hunk 2]
```diff
@@ -20,8 +20,13 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Replcae Docker tag
```
Comment (payneBrandon) [inline .github/workflows/dockerhub.yml:24]:
```suggestion
      - name: Replace Docker tag
```

Comment (dmccoystephenson) [reply .github/workflows/dockerhub.yml]: Applied this suggestion!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**